app/models/jobstatusdata.rb

```diff
@@ -27,6 +27,7 @@ def initialize(info, cluster=OODClusters.first.id.to_s, extended=false)
     self.account = info.accounting_id
     self.status = status_label(info.status.state.to_s)
     self.cluster = cluster
+    self.cluster_title = build_title(cluster)
```

Comment:
I'd argue that that's actually an issue with the other code and not this one. I guess it *should* always be available, but I guess it depends on how defensive we want to make the programming here.